internal/k8s/watch.go

```diff
@@ -317,16 +337,24 @@ func (kCli K8sClient) WatchEverything(ctx context.Context, lps []model.LabelPair
 	}
 
 	var watchers []watch.Interface
+	fmt.Printf("🔎 found %d watchable gvrs\n", len(gvrs))
```

Comment:
👀

Reply:
ALSO SUPPOSED TO BE THERE WHAT ARE YOU TALKING ABOUT